docs-2.0/5.configurations-and-logs/1.configurations/3.graph-config.md

```diff
@@ -99,6 +99,12 @@ For all parameters and their current values, see [Configurations](1.configuratio
 | ------------------- | ------------------------ | ------------------------------------------ |
 | `system_memory_high_watermark_ratio` | - | Specifies the trigger threshold of the high-level memory alarm mechanism. The default value is `0.9`. If the system memory usage is higher than this value, an alarm mechanism will be triggered, and Nebula Graph will stop querying. This parameter is not predefined in the initial configuration files. |
 
+## Metrics configurations
+
+| Name | Predefined value | Description |
+| - | - | - |
+| `enable_space_level_metrics` | `false` | Enable or disable space-level metrics. Such metric names include the graph space name that it monitors, for example, `query_latency_us{space=basketballplayer}.avg.3600`. You can view the supported metrics with the `curl` command. For more information, see [Query Nebula Graph metrics](../../6.monitor-and-metrics/1.query-performance-metrics.md). |
```

Comment:
from the example, query_latency_us{space=basketballplayer}.avg.3600, i guess, it is that a space-level metric name should contain the corresponding graph space name. 

1. we can use "contain" instead of "include" when specifying an object has something inside it.
2. it monitors a graph space rather than a graph space name.
please correct me if I'm wrong.

Reply:
1. In this situation, they are synonymous. 
2. Ur right. I'll change it to the name of the graph space